appendices/spirvenv.txt

```diff
@@ -255,8 +255,10 @@ endif::VK_VERSION_1_1[]
     [eq]#v# components
   * [[VUID-{refpage}-OpTypeImage-04661]]
     Objects of types code:OpTypeImage, code:OpTypeSampler,
-    code:OpTypeSampledImage, and arrays of these types must: not be stored
-    to or modified
+    code:OpTypeSampledImage, code:OpTypeAccelerationStructureKHR,
+    and arrays of these types must: not be stored to or modified
```

Comment:
Does this need a new VUID since it is effectively collapsing 04661 and 04665 into a single VU statement?

Reply:
The reason to give a new VUID has always been to notify downstream of the change to the validation implementors... but as posted in the summary, this hasn't been implemented yet and I plan to personally do it, so see no reason to change the VU